package.json

```diff
@@ -35,14 +38,13 @@
     "name": "FEC",
     "url": "https://www.fec.gov/"
   },
-  "licenses": [
-    {
-      "type": "CC0-1.0",
-      "url": "http://github.com/fecgov/fec-eregs/blob/master/LICENSE.md"
-    }
-  ],
+  "license": "ISC",
```

Comment:
Is this the correct license information for eregs?

Reply:
permissive free software license.  I think it's appropriate.

Reply:
Those styles are now deprecated. So i used the latest syntax.

Reply:
This doesn't match the license information for FEC projects, which is a combination of things but mostly CC0. https://github.com/fecgov/FEC/blob/master/LICENSE.md

Reply:
Reasons to remove the licenses tag (see #https://docs.npmjs.com/files/package.json#license)
1. Its deprecated so i replaced with license
2. Deprecated warning related to licenses will not show up now
3. I had to recreate package.json file (npm init).
4. We already have fec-eregs license info defined in License.md file.

Reply:
CCO licenses got added 5 months ago on fec-eregs repo. We didn't have one prior that.  What is this CC0 licence mean? Not having this license really doesn't make any difference with the way the fec-eregs repo work.  I would keep things more simple and easy to understand and delete anything we don't need.

More info on ISC license:
https://choosealicense.com/licenses/isc/

Reply:
Ok, I'm fine with removing the license tag but we shouldn't put the wrong license in this file.

Reply:
To be clear, `licenses` was deprecated in favor of `license`.

If it matters, we're using
`"license": "CC0-1.0",`
for the cms and api

Reply:
ISC or CC0 both are just standards to defining license. What is the `wrong` information that you are referring to in package.json file?